### PR TITLE
Yg fs4 bio/hero

### LINKED ISSUE #39: yg-fs4-bio
Bio will have a brief disrcription of who I am and a bit of my coding style.

### PR DESCRIPTION
**Pull request by yadetagetachew**
#Changes
List Changes Introduced by this PR
*hero with introduction of me
*about me photo and lorem as placeholder added

#Purpose
Present my name and a snippet describing myself

#Approach
created hero on canva 

Closes #39

### CODE REVIEW

**Comment by yadetagetachew** [comment]
accidentally worked out of scope and added hero section as well

**Review by Tabspear — APPROVED** [review]

File: yadi/yadi_personal.html [hunk 1]
```diff
@@ -16,12 +16,77 @@
...
-        <a href="/index.html"><img src="/yadi/yadi_img/yg-logo-white.png" alt="Home"></a>
-        <a href="#about">About</a>
-        <a href="#services">Projects</a>
-        <a href="#contact">Contact</a>
+        <a href="/index.html"><img src="/yadi/yadi_img/yg-logo.gif" alt="Home"></a>
+        <a id="navigation" href="#about">About</a>
+        <a id="navigation" href="#services">Projects</a>
+        <a id="navigation" href="#contact">Contact</a>
     </div>    
+
```
Comment (gabymoreno9) [inline yadi/yadi_personal.html:61]: Example review comment here

File: yadi/yadi_personal.html [hunk 2]
```diff
@@ -16,12 +16,77 @@
     <title>Yadi</title>
 </head>
 <body>
+
+    <!-- <nav class="navbar navbar-dark bg-transparent fixed-top w-100">
```
Comment (gabymoreno9) [inline yadi/yadi_personal.html:20]: Good job here! Or something

File: yadi/yadi_personal.html [hunk 3]
```diff
@@ -16,12 +16,77 @@
...
-        <a href="#contact">Contact</a>
+        <a href="/index.html"><img src="/yadi/yadi_img/yg-logo.gif" alt="Home"></a>
+        <a id="navigation" href="#about">About</a>
+        <a id="navigation" href="#services">Projects</a>
+        <a id="navigation" href="#contact">Contact</a>
     </div>    
+
+    <div class="hero d-flex justify-content-center ">
+        <img class="img-fluid me-4" src="/yadi/yadi_img/hero.png">
+        <!-- <h1 class="d-flex justify-content-center align-self-center">Hey, <br>I'm <br> Yadi G. -->
```
Comment (gabymoreno9) [inline yadi/yadi_personal.html:64]: Remove this or something

**Pull request merged**